test/mongodb.test.js

```diff
@@ -211,7 +220,7 @@ describe('mongodb connector', function() {
         /* eslint-disable camelcase */
         var indexes =
         { _id_: [['_id', 1]],
-          geometry_2dsphere: [['geometry', '2dsphere']],
+          geojson_location_geometry: [['location.geometry', '2dsphere']],
```

Comment:
I'm pretty sure it still does, I basically changed the name of the key. The key is still created with the right type.